codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaders.java

```diff
@@ -1687,4 +1687,20 @@ public final String getAsString(CharSequence name) {
     public boolean contains(CharSequence name, CharSequence value, boolean ignoreCase) {
         return contains(name.toString(), value.toString(), ignoreCase);
     }
+
+    @Override
+    public String toString() {
```

Comment:
@slandelle did you consider overriding this in `DefaultHttpHeaders` and have special implementation that not needs to allocate an extra List etc ?

Reply:
My intent was solely for debugging, so I went with the most straightforward implementation. Will change to use the iterator.